src/codegen/enums.rs

```diff
@@ -459,6 +459,25 @@ impl FromGlib<{sys_crate_name}::{ffi_name}> for {name} {{
         )?;
         writeln!(w)?;
 
+        version_condition(w, env, None, version, false, 0)?;
+        cfg_condition_no_doc(w, config.cfg_condition.as_ref(), false, 0)?;
+        allow_deprecated(w, enum_.deprecated_version, false, 0)?;
+        writeln!(
+            w,
+            "impl {glib}HasParamSpec for {name} {{
+                type ParamSpec = {glib}ParamSpecEnum;
+                type SetValue = Self;
+                type BuilderFn = fn(&str, Self) -> {glib}ParamSpecEnumBuilder<Self>;
+    
+                fn param_spec_builder() -> Self::BuilderFn {{
+                    |name, default_value| Self::ParamSpec::builder_with_default(name, default_value)
+                }}
+}}",
+            name = enum_.name,
+            glib = use_glib_type(env, ""),
```

Comment:
```suggestion
            glib = use_glib_type(env, "HasParamSpec"),
```

or not? And then remove the suffix above